include/alpaka/platform/PlatformUniformCudaHipRt.hpp

```diff
@@ -134,8 +134,8 @@ namespace alpaka
             {
                 ALPAKA_DEBUG_FULL_LOG_SCOPE;
 
-                std::size_t const kiB(1024);
-                std::size_t const miB(kiB * kiB);
+                std::size_t constexpr kiB(1024);
+                std::size_t constexpr miB(kiB * kiB);
```

Comment:
Since you're touching these lines anyway:

```suggestion
                constexpr auto KiB = std::size_t{1024};
                constexpr auto MiB = KiB * KiB;
```

Also the renaming from `{k,m}iB` to `{K,M}iB` needs to be applied where necessary.

Reply:
This should be included now.